### PR TITLE
Fixes bsc#1089938

### PR DESCRIPTION
**Pull request by pcacjr**

### CODE REVIEW

File: src/modules/SambaAD.pm [hunk 1]
```diff
@@ -122,13 +125,12 @@ sub GetMachines {
 # @return string	non empty when ADS was found
 BEGIN{$TYPEINFO{GetADS}=["function","string","string"]}
 sub GetADS {
-
```
Comment (dmulder) [inline src/modules/SambaAD.pm]: Why the whitespace changes?

File: src/modules/SambaAD.pm [hunk 2]
```diff
@@ -49,6 +49,9 @@ use constant {
     FALSE => 0,
 };
 
+# create a dummy smb.conf file only for performing the libnet commands
+my $dummy_conf_file = SCR->Read (".target.tmpdir") . "/dummy-smb.conf";
+SCR->Write (".target.string", $dummy_conf_file, "");
```
Comment (dmulder) [inline src/modules/SambaAD.pm]: I don't see this being cleaned up.
Also, there's a potential attack vector here. What if an attacker pre-fills /tmp/dummy-smb.conf with malicious settings (is that where ".target.tmpdir" points to?). You should also check the permissions on the file (make sure it's not world writable). Is there an equivalent to tmpfile() in perl?

File: package/yast2-samba-client.changes [hunk 3]
```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Fri Apr 27 13:36:58 UTC 2018 - palcantara@suse.de
+
+- yast-samba-client fails to join if /etc/samba/smb.conf or
+  /etc/krb5.conf don't exist; (bsc#1089938)
+
```
Comment (teclator) [inline package/yast2-samba-client.changes]: Version is missing in the changelog, please add it.

Comment (teclator) [reply package/yast2-samba-client.changes]: BTW I have not merged #60 so I can adapt the entry anyway.

**Pull request merged**